### PR TITLE
Add steam-locomotive

### PR DESCRIPTION
**Pull request by 85pando**

### CODE REVIEW

File: pages/common/sl.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# sl
+
+> This command is normaly invoked, when you mistype `ls` and provides you with a steam locomotive running through your terminal.
```
Comment (igorshubovych) [inline pages/common/sl.md]: Description is too long.

**Comment by 85pando** [comment]
Fixed

**Comment by igorshubovych** [comment]
How about just

```Markdown
> Steam locomotive running through your terminal
```

**Comment by 85pando** [comment]
I think I have to further get into the minimalist mindset required here (-;

**Comment by igorshubovych** [comment]
LGTM.

**Comment by felixonmars** [comment]
:+1:

**Pull request merged**